src/api/index.js

```diff
@@ -18,3 +18,13 @@ export const getUserInfo = () => logined.get('user/my');
 
 export const signUp = ({ account, password, find_email, nickname }) =>
   logined.post('/user/sing-in', { account, password, find_email, nickname });
+
+
+export const keywordAPI = {
+  InquiryKeyword : () => logined.get(`/keyword`),
+  getKeywordList : (keywordName) => logined.get(`/keyword/list?keyword-name=${keywordName}`),
+  deleteKeywordList : (query) => logined.patch(`/keyword/list/notice?${query}`),
+  deleteKeywordItem : (id) => logined.patch(`/keyword/list/notice?notice-id=${id}`),
+  movekeywordList : (data) => logined.post(`/scrap`,{"board_id":data}),

```

Comment:
해당 코드를 실행시켜본건 아니라 어떤 문맥에서 post /scrap 요청이 `moveKeywordList` 이 되는 지 모르겠네요
`addScrap` 정도면 되지 않을까요?

```suggestion
  addScrap : (data) => logined.post(`/scrap`,{"board_id":data}),

```

Reply:
post/scrap 요청이 보관함으로 이동시키는 것이여서 movekeywordList로 네이밍 했습니다.
addScrap으로 수정하도록 하겠습니다!

Reply:
9f14a7e

Reply:
음... 참고로 저는 앞에 접두어를 `create (post) / get / update(put) / delete` 로 통일해서 사용하고자 합니다 
`add` 도 나쁘지 않고요 :) 개발하시면서 상윤님만의 네이밍 원칙을 가지시는 것도 좋겠네요~